Makefile

```diff
@@ -4,6 +4,10 @@ export COMPOSE_DOCKER_CLI_BUILD=1
 export DOCKER_BUILDKIT=1
 export COMPOSE_PROFILES=local
 
+define random
+$(shell cat /dev/urandom | LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
Why the change from pwgen, and why is it in this PR?

I'm pretty sure we've had the conversation before about not including extra things in PRs, as it slows down the review process. :frowning:

Reply:
pwgen is not a default tool, this approach should work out of box on linux and on mac
without this change i cannot test locally

Reply:
Interesting.  Well, it does seem to work ok, and the quality of randomness doesn't really make a difference for development.

The `pwgen` change should really be in its own PR though, as it's a distinct thing.